content/en/blog/general/apply-vap-using-cli/index.md

```diff
@@ -0,0 +1,222 @@
+---
+date: 2023-09-04
+title: "Applying Validating Admission Policies using Kyverno CLI"
+linkTitle: "Applying Validating Admission Policies using Kyverno CLI"
+author: Mariam Fahmy
+description: "Using Kyverno CLI to apply Validating Admission Policies"
+---
+The [Kyverno Command Line Interface (CLI)](https://kyverno.io/docs/kyverno-cli/) is designed to validate and test policy behavior to resources prior to adding them to a cluster.
```

Comment:
Because this blog post is on the same site as the docs, when you reference links on this site, you omit the hostname portion so all that's needed for this link is "/docs/kyverno-cli/".

Reply:
Thanks for the info.